explainer-client/src/main/scala/ExplainEditorJSDomBuilders.scala

```diff
@@ -23,10 +23,20 @@ import scala.scalajs.js.JSON
 object ExplainEditorJSDomBuilders {
 
   def statusBarText(explainer: CsAtom) = {
-    val isPublished = explainer.contentChangeDetails.published.isDefined
 
-    if(isPublished) {
-      "published"
+    def getLongFromOptionCSChangeRecordForComparison(optionCSChangeRecord : scala.Option[CsChangeRecord]) : Long = {
+      optionCSChangeRecord match {
+        case Some(cr) => cr.date
+        case None => 0L
+      }
+    }
+    
+    if(explainer.contentChangeDetails.published.isDefined) {
+      if ( getLongFromOptionCSChangeRecordForComparison(explainer.contentChangeDetails.published) < getLongFromOptionCSChangeRecordForComparison(explainer.contentChangeDetails.lastModified) ){
+        "unseen"
+      }else{
+        "published"

```

Comment:
We should ultimately rewrite the client side of this logic in Scala JS and change this to using a typed enum rather than strings.